### PR TITLE
Adding Spanish translation

### PR DESCRIPTION
**Pull request by rasputino**
# Spanish translation

Added resources and .vsct file for Spanish language.

## Type of change

- [x ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

With the Visual Studio debugger.

### CODE REVIEW

**Comment by rasputino** [comment]
@ErikEJ I could not test the contextual menu with the debugger although the vsct file has been added, how could I check it before changing the state of the pull request to ready?

image

**Comment by ErikEJ** [comment]
@rasputino I will get the PR and investigate the missing menu translation.

**Comment by ErikEJ** [comment]
@rasputino and thanks, this is incredibly useful! Spanish is a very used language

**Comment by ErikEJ** [comment]
@rasputino  I cannot verify either, suggest you change to real PR and I will approve, and we can fix any issues as a follow up

**Comment by ErikEJ** [comment]
@rasputino I latest daily build soon, are you able to test the context menu?

File: src/GUI/EFCorePowerTools/EFCorePowerTools.csproj [hunk 1]
```diff
@@ -447,6 +455,9 @@
     <VSCTCompile Include="EFCorePowerToolsPackage.zh-Hans.vsct">
       <ResourceName>Menus.ctmenu</ResourceName>
     </VSCTCompile>
+    <VSCTCompile Include="EFCorePowerToolsPackage.es.vsct" >
```
Comment (rasputino) [inline src/GUI/EFCorePowerTools/EFCorePowerTools.csproj:458]: Changes to add Spanish resources to contextual menu

**Comment by rasputino** [comment]
@ErikEJ No, the changes in contextual menu are "still" in English:

image

The changes in the dialogs worked.

image

Version used:

image

It seems that the csproj changes are not enough to make the resource file be used:

https://github.com/ErikEJ/EFCorePowerTools/pull/1840/files#r1246376335

**Comment by ErikEJ** [comment]
That is very odd. Wonder what we are doing wrong? @madskristensen

**Pull request merged**